Bottom: 30f08dec763d02b7ef199920aa6571d25bf03aaa
Top:    b74db24802ab41983c854ec8aa16ed715e976e2d
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-05-05 16:54:49 +0530

Refresh of sched-core-introduce-per_cpu

---

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 3c49b839ff24..4ec4cef13088 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -6647,8 +6647,6 @@ void __init sched_init(void)
 			cpumask_size(), GFP_KERNEL, cpu_to_node(i));
 		per_cpu(select_idle_mask, i) = (cpumask_var_t)kzalloc_node(
 			cpumask_size(), GFP_KERNEL, cpu_to_node(i));
-		per_cpu(nr_lat_sensitive, i) = (cpumask_var_t)kzalloc_node(
-			cpumask_size(), GFP_KERNEL, cpu_to_node(i));
 	}
 #endif /* CONFIG_CPUMASK_OFFSTACK */
